.travis.yml

```diff
@@ -1,56 +1,75 @@
 language: cpp
 sudo: false
-before_install:
-        - export LD_LIBRARY_PATH=`pwd`/tests/:$LD_LIBRARY_PATH
 before_script:
-        - wget https://github.com/groundx/capstonefuzz/raw/master/corpus/corpus-libFuzzer-capstone_fuzz_disasmnext-latest.zip
-        - unzip -q corpus-libFuzzer-capstone_fuzz_disasmnext-latest.zip -d suite/fuzz
-        # TODO remove built in cmocka compile and use system cmocka (including brewfile) once xenial is default
-        - git clone https://git.cryptomilk.org/projects/cmocka.git suite/cstest/cmocka
-        - chmod +x suite/cstest/build_cstest.sh
+  - wget https://github.com/groundx/capstonefuzz/raw/master/corpus/corpus-libFuzzer-capstone_fuzz_disasmnext-latest.zip
+  - unzip -q corpus-libFuzzer-capstone_fuzz_disasmnext-latest.zip -d suite/fuzz
```

Comment:
this PR is for master branch, but above lines are about next branch? or do you actually want to update the next branch?

Reply:
I based this off of `master`, didn't know about the `next` branch